drivers/hmis/app/models/hmis/ce/match/client_field_map.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
-# resolve fields from the client table
+# resolve fields from the client, enrollment, and project table
+#
+# - Question: should we move Enrollment/Project/Referral-related fields to another FieldMap? They require additional joins
```

Comment:
Perhaps? I think I'd be inclined to keep them in the same resolver for now. Maybe the shape of an additional resolvers be clear from the full list of fields.